docs/release-notes/self-hosted/self-hosted-v5.0.0.md

```diff
@@ -0,0 +1,93 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v5.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/3.9.16
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/4.0.28
+---
+
+# Self-hosted v5.0.0
+
+These release notes are for [Codacy Self-hosted v5.0.0](https://github.com/codacy/chart/releases/tag/5.0.0){: target="_blank"}, released on December 14, 2021. <!-- TODO Update release date -->
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Breaking changes
+
+This version of Codacy Self-hosted no longer includes support for legacy manual organizations.
```

Comment:
Do we know if we have customers using Codacy Self-hosted with legacy manual organizations? And if so, what should be the process to migrate those organizations before upgrading to the new version?